src/main/webapp/components/utils/getField.js

```diff
@@ -0,0 +1,12 @@
+/**
+  * Returns the value at obj[field] or defaultValue
+  * if obj[field] === undefined.
+  * Useful for extracting a field from an object, with a fallback.
+  *
+  * @param obj the object to query the fields for.
+  * @param field the target field, in string form. 
+  * @param defaultValue the value to be used if obj[field] === undefined.
+ */
+export const getField = (obj, field, defaultValue) => obj[field] !== undefined
+? obj[field]
```

Comment:
the indentation here seems a bit odd. I would suggest changing to `obj[field] !== undefined ?` on a separate line, with [proper indentation](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation), followed by `obj[field] : ....` on the next line, at the same level of indentation.

Reply:
I suggest to just use go/google-java-format and not worry too much about formatting.

Reply:
Added {} pair to the function body, and used a return statement instead. Split the ternary operator according to [4.5.1](https://google.github.io/styleguide/jsguide.html#formatting-where-to-break)